app/Models/SocialiteUser.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class SocialiteUser extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $guarded = [
+        'id',
+    ];
+
+    public function user(): BelongsToMany
+    {
+        return $this->belongsToMany(User::class);
```

Comment:
I'm surprised this is a belongsToMany relationship, can one socialite user have many users? The migration file indicates that the socialite user belongsTo a user.

Reply:
I was thinking about this. You are right, should be 1-1 relationship